plugins/urls/__init__.py

```diff
@@ -293,7 +317,7 @@ def urls_command(self, protocol, caller, source, command, raw_args,
                     source.name: {
```

Comment:
This section seems like it could do with a refactor. The default dict is here twice. I also don't like how the shortener is  set per channel merely by running the command. If the default is changed in future, all channels where this has been run before will still have the old one.